### PR TITLE
V2.17-RUN-16507-dashboards-mvp

### PR DESCRIPTION
**Pull request by jasonnovichRunAI**

### CODE REVIEW

**Comment by jasonnovichRunAI** [comment]
@lavianalon even though I have put this PR in, there is still a lot missing. Need a CPU dashboard and working multi-cluster dashboard to expand this more.

File: docs/admin/admin-ui-setup/dashboard-analysis.md [hunk 1]
```diff
@@ -2,95 +2,106 @@
 
 The Run:ai Administration User Interface provides a set of dashboards that help you monitor Clusters, Cluster Nodes, Projects, and Workloads. This document provides the key metrics to monitor, how to assess them as well as suggested actions.
 
+Dashboards are used by system administrators to analyze and diagnose issues that relate to:
```
Comment (lavianalon) [inline docs/admin/admin-ui-setup/dashboard-analysis.md:5]: All this new section is related to the new Overview. So it should be under the overview dashboard in line 13

File: docs/admin/admin-ui-setup/dashboard-analysis.md [hunk 2]
```diff
@@ -2,95 +2,106 @@
...
 There are 5 dashboards:
 
-* [**Overview**](#overview-dashboard) dashboard&mdash;Provides information about what is happening right now in the cluster.
+* [**GPU/CPU Overview**](#gpucpu-overview-dashboard) dashboard&mdash;Provides information about what is happening right now in the cluster.
 * [**Quota Management**](#quota-management-dashboard) dashboard&mdash;Provides information about quota utilization.
 * [**Analytics**](#analytics-dashboard) dashboard&mdash;Provides long term analysis of cluster behavior.
 * [**Multi-Cluster Overview**](#multi-cluster-overview-dashboard) dashboard&mdash;Provides a more holistic, multi-cluster view of what is happening right now. The dashboard is intended for organizations that have more than one connected cluster.
 * [**Consumption**](#consumption-dashboard) dashboard&mdash;Provides information about resource consumption.
 
-## Overview Dashboard
```
Comment (lavianalon) [inline docs/admin/admin-ui-setup/dashboard-analysis.md:13]: Should be "New Overview dashboard"

File: docs/admin/admin-ui-setup/dashboard-analysis.md [hunk 3]
```diff
@@ -2,95 +2,106 @@
...
-
-The Overview dashboard provides information about what is happening **right now** in the cluster.  Administrators can view high-level information on the state of the cluster, including:
+## GPU/CPU Overview Dashboard
```
Comment (lavianalon) [inline docs/admin/admin-ui-setup/dashboard-analysis.md:34]: Should add "Legacy"

File: docs/admin/admin-ui-setup/dashboard-analysis.md (hunk 1 above)
Comment (jasonnovichRunAI) [reply docs/admin/admin-ui-setup/dashboard-analysis.md:5]: Actually this is more a general description of all the dashboards and should conceptually hold true for all of them. If there are dashboards that are not used by Admins, meaning like a researcher or other then I can add those roles to sentence.

File: docs/admin/admin-ui-setup/dashboard-analysis.md (hunk 3 above)
Comment (jasonnovichRunAI) [reply docs/admin/admin-ui-setup/dashboard-analysis.md:34]:
```suggestion
## GPU/CPU Overview Dashboard (New and legacy)
```

File: docs/admin/admin-ui-setup/dashboard-analysis.md (hunk 2 above)
Comment (jasonnovichRunAI) [reply docs/admin/admin-ui-setup/dashboard-analysis.md:13]:
```suggestion
## New Overview Dashboard
```

**Pull request merged**